config/foreman-answers.yaml

```diff
@@ -12,6 +12,7 @@
 foreman:
   custom_repo: true
   email_config_method: database
+  dynflow_in_core: true
```

Comment:
@dLobatog the current puppet code defaults to true in params.pp and like @mmoll pointed out: this won't end up in 1.14 so I don't think this is actually needed.